src/pages/Profile.tsx

```diff
@@ -77,6 +77,12 @@ const styles = {
   },
   personalInfoContent: { display: "flex", gap: 0.5 },
   profileContent: { padding: 2 },
+  tab: {
+    ":hover": {
+      backgroundColor: "primary.light",
+    },
+    transitionDuration: "0.25s",
```

Comment:
another implementation of it
```suggestion
    transition: "0.25s ease",
```